lld/COFF/Writer.cpp

```diff
@@ -1156,6 +1162,11 @@ void Writer::createSections() {
     sec->addContributingPartialSection(pSec);
   }
 
+  if (ctx.hybridSymtab) {
+    if (OutputSection *sec = findSection(".CRT"))
```

Comment:
Oh, tricky. I guess we'd need to do something similar for every section that contains sorted ranges of pointers like this - presumably `.ctors` too?

What about sections like `.tls`? I presume that's coming up in a future patch too :-)